source/mainnet/net/guides/dapp-examples.rst

```diff
@@ -61,7 +61,7 @@ Select an example to see more information about it, such as a hosted dApp for yo
 
     `Demo front end <https://voting.testnet.concordium.com>`__ where you can try the functionality on Concordium's testnet
 
-    :ref: `Tutorial about the Voting dApp<voting-dapp>`
+    :ref:`Tutorial about the Voting dApp<voting-dapp>`
```

Comment:
Is this intended?

Reply:
Yes, the link was broken because of the extra whitespace. You can see similar links in the same file which all have NO whitespace.